src/app/models/groups.ts

```diff
@@ -0,0 +1,24 @@
+export interface Groups {
+
+  name: string,
```

Comment:
Inconsistency in al interface. in some interface semicolon is used and in some comma is used